Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1637,6 +1638,49 @@ def get_mapping_fields_command(search_base):
 '''
 
 
+def set_password_not_expire(default_base_dn):
+    args = demisto.args()
+    sam_account_name = args.get('username')
+    pwd_n_exp = args.get('value')
+
+    # query by sAMAccountName
+    if sam_account_name or args.get('sAMAccountName'):
+        if sam_account_name:
+            username = escape_filter_chars(sam_account_name)
+        else:
+            username = escape_filter_chars(args['sAMAccountName'])
+        query = "(&(objectClass=User)(objectCategory=person)(sAMAccountName={}))".format(username)
+
+    entries = search_with_paging(query, default_base_dn, attributes='userAccountControl', size_limit=0, time_limit=0)
+    user = entries.get('flat')[0]
+    user_account_control = user.get('userAccountControl')[0]
+
+    # Check if UAC flag for "Password Never Expire" (0x10000) is set to True or False:
+    if pwd_n_exp == 'true':
+        # Sets the bit 16 to 1
+        user_account_control |= 1 << 16
+        content_output = f"AD account {username} has set \"password never expire\" attribute. Value is set to True"
+    else:
+        # Clears the bit 16 to 0
+        user_account_control &= ~(1 << 16)
+        content_output = f"AD account {username} has cleared \"password never expire\" attribute. Value is set to False"
+
+    attribute_name = 'userAccountControl'
+    attribute_value = user_account_control
+    dn = user_dn(sam_account_name, default_base_dn)
```

Comment:
The 'sam_account_name' variable refers to the 'username'. 
What happens in cases where the user used the 'sAMAccountName' and not the 'username'?